api.md

```diff
@@ -170,6 +171,114 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## add_data_node() :community_function: [](add_data_node)
```

Comment:
I do not believe licensing has been finalized.  Would like to pause merging (although can fix below) until finalized, or possibly just remove the decorator until we do (will get back to you today about what to do here).  Sorry for mixed messages. :)

Reply:
Ok. Sitting on the commit for the time being then.

Reply:
From what I understand, the decision was that the decorator should not be there, so removing it.